src/elements/content-sidebar/__tests__/MetadataSidebarRedesign.test.tsx

```diff
@@ -0,0 +1,90 @@
+import React from 'react';
+import { userEvent } from '@testing-library/user-event';
+import { FIELD_PERMISSIONS_CAN_UPLOAD } from '../../../constants';
+import { screen, render } from '../../../test-utils/testing-library';
+
+import {
+    MetadataSidebarRedesignComponent as MetadataSidebarRedesign,
+    type MetadataSidebarRedesignProps,
+} from '../MetadataSidebarRedesign';
+
+describe('elements/content-sidebar/Metadata/MetadataSidebarRedesign', () => {
```

Comment:
Don't we need tests for the errorMessage and loading indicator?

Reply:
added ✨